src/components/Questions.jsx

```diff
@@ -1,25 +1,44 @@
 import React, { useState, useEffect } from "react";
 import Axios from 'axios';
+import { connect } from 'react-redux';
+import {
+  Link,
+} from 'react-router-dom';
+import BorderTopCard from './BorderTopCard';
 
-function Questions({ props }) {
-  const [question, setQuestion] = useState(null);
+function Questions(props) {
+  const [questions, setQuestion] = useState();
```

Comment:
you are here starting with undefined. start with an empty array:

    const [questions, setQuestion] = useState([]);

Reply:
j'ai miss click